presto-main/src/main/java/com/facebook/presto/resourcemanager/ResourceManagerClient.java

```diff
@@ -42,4 +42,7 @@ List<ResourceGroupRuntimeInfo> getResourceGroupInfo(String excludingNode)
 
     @ThriftMethod
     void resourceGroupRuntimeHeartbeat(String node, List<ResourceGroupRuntimeInfo> resourceGroupRuntimeInfo);
+
+    @ThriftMethod
+    Integer getRunningTaskCount();
```

Comment:
nit : do we need the boxed type here?. does primitive int suffice since that is what your implementation in [ResourceManagerClusterStateProvider.java](https://github.com/prestodb/presto/pull/18732/files#diff-418d074b5a5a3af8c53df4de42e75e46b8e3fff07b2bccde12608225448190f9) returning.